Dockerfile

```diff
@@ -1,16 +1,129 @@
-FROM ubuntu:18.04
-MAINTAINER datapunt@amsterdam.nl
+FROM ubuntu:22.04
+LABEL maintainer="datapunt@amsterdam.nl"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV TZ Europe/Amsterdam
+
+RUN apt-get -y update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gettext \
+        xz-utils \
+        cmake \
+        gcc \   
+        g++ \
+        libfreetype6-dev \
+        libglib2.0-dev \
+        libcairo2-dev \
+        git \
+        locales \
+        make \
+        patch \
+        openssh-server \
```

Comment:
Why are we installing an SSH server?

Reply:
We can do without, I suppose. @lennardstam ?

Reply:
sure, package is removed from the list